Improvements to bring the homepage

lhmarsden opened the pull request:
I have made some improvements to the homepage
- Option to add a hero image or gradient
- Reformatting some of the sections to improve appearence
- Inclusion of photo from 2024 workshop in the 'Get involved' section.

<img width="1850" height="1905" alt="screencapture-127-0-0-1-4000-2025-09-25-14_17_41" src="https://github.com/user-attachments/assets/1ec45eb8-3558-4dd2-91ce-24266e0b5d54" />

lhmarsden commented on index.md:
```diff
@@ -3,33 +3,50 @@ layout: default
+
+### Check the latest vocabularies
+
+<a href="Data/cf-standard-names/current/build/cf-standard-name-table.html" style="display:inline-block; background-color:#3498db; color:white; padding:14px 22px; border-radius:6px; text-decoration:none; font-weight:bold; font-size:1.6rem; margin-right:8px; margin-bottom:8px;">Standard names</a>
```
If I do another commit I will fix this

DocOtak replied:
Looks good now

lhmarsden commented:
> Was this agreed in the hackathon yesterday?

We just agreed that someone needs to just do something since there had not been any activity. This was also communicated in the summary of the hackathon afterwards. 

The 'hero image' was agreed in last year's workshop. The black box is just an equivalent which can be overwritten should we want an image there instead. 

There was also a demand in the workshop to make the links stand out more - to 'buttonise' them, if you will.

> Usually we open issues for substantial enhancement or correction of defects on the web page. I think it would be good to have an issue for this, with this PR linked to it (like we do for conventions).

I have created one now https://github.com/cf-convention/cf-convention.github.io/issues/606

> I feel that the black box at the top of the page is too tall. It seems to dominate.

In the hackathon we were joined by Erik Kusch from the Darwin Core community and he shared what they came up with in a recent website update. It was similar to this, and I took this as inspiration. See below.
https://dwc.tdwg.org/

I personally like this approach. It is quite commonplace for websites to have a big image/video/carrousel of images with the websites key 'call to action' overlain on it. The scale is important because it draws people to the call to action. We could replace the solid background with an image - but then we would need a suitable image.

If the community doesn't like this approach I can certainly make it a bit smaller - but I do feel it should be larger than the content, so to draw people in to the 'call to action'. 

> And a lighter colour?

I chose this colour because it is the same as the navbar. It we change this, the navbar colour should be changed too, or it will look weird.

> The photo also may be a bit larger than necessary, I feel, and it would be good to give it a caption.

This can be changed.

Let me know how you want me to proceed on this. I can make the proposed changes in the meantime.

lhmarsden commented:
<img width="1850" height="1642" alt="screencapture-127-0-0-1-4000-2025-09-26-13_23_53" src="https://github.com/user-attachments/assets/0eb4ebbb-b88e-4d5c-9812-599b424691b3" />

lhmarsden commented:
Without the flash message now since the workshop has finished.

<img width="1850" height="1592" alt="screencapture-127-0-0-1-4000-2025-09-26-13_27_14" src="https://github.com/user-attachments/assets/9a26ba37-7ab3-4e09-b8b2-feb3e7c89089" />

JonathanGregory commented:
Dear Luke

Thanks for these improvements. The bar at the top looks a reasonable size now. It's still a bit overbearing, and that's because of the colour, I think. As far as I know, there's no reason for the navigation bar to be black. Any lighter colour would be preferable!

A minor point on content: What about having a button for the standard name table in the coloured part, next to the latest release of the conventions? I guess many visitors are looking for that.

Two even more minor points on aesthetics:

To my eye, the use of red and green colours for buttons is distracting, because it doesn't have an obvious meaning. I would make all the buttons on the page the same colour, unless we have a distinction to draw.

For balance, I think there ought to be a horizontal line between the "Get involved" and "Quick links" sections.

Cheers

Jonathan

lhmarsden commented:
> The bar at the top looks a reasonable size now. It's still a bit overbearing, and that's because of the colour, I think. As far as I know, there's no reason for the navigation bar to be black.

I personally like the darker colour, but okay. I have changed it to a cool blue.

I also fixed both minor points.

> A minor point on content: What about having a button for the standard name table in the coloured part, next to the latest release of the conventions? I guess many visitors are looking for that.

This could be done. Could be a development later? I won't have time for this next week. I would prefer this to be merged if it is an improvements and we can develop iteratively. 

But I agree it would be an improvement for it to be there too. 

My latest effort:

<img width="1850" height="1613" alt="screencapture-127-0-0-1-4000-2025-09-26-15_56_57" src="https://github.com/user-attachments/assets/771ccaa8-adfc-4166-a82d-0b3e4a688132" />
